### ISSUE TITLE
 does not recognize the uri

### ISSUE THREAD

**Issue by michaellcader** [BishopFox/jsluice#6]
```-package main

import (
	"encoding/json"
	"fmt"

	"github.com/BishopFox/jsluice"
)

func main() {
	analyzer := jsluice.NewAnalyzer([]byte(`
	function C() {
        return C = (0, o.default)(c().mark(function e(t) {
                return c().wrap(function (e) {
                    while (1)
                        switch (e.prev = e.next) {
                        case 0:
                            return e.abrupt("return", (0, i.default)("/admin/article/paging", {
                                    method: "POST",
                                    body: t
                                }));
                        case 1:
                        case "end":
                            return e.stop()
                        }
                }, e)
            })),
        C.apply(this, arguments)
    }
	`))

	for _, url := range analyzer.GetURLs() {
		j, err := json.MarshalIndent(url, "", "  ")
		if err != nil {
			continue
		}

		fmt.Printf("%s\n", j)
	}
}

```

 does not recognize the uri:

"/admin/article/paging"

**Comment by bf-tomnomnom** [comment]
Hey @michaellcader, thanks for raising an issue!

This is related to the logic that `jsluice` uses to identify paths. At the moment the [MaybeURL](https://github.com/BishopFox/jsluice/blob/main/maybeurl.go) function is quite dependent on there being a file extension, query string, or something else like that to increase confidence.

I've been thinking about this for a while and think there are some tweaks we can do to improve the situation without catching too many false positives.

I'll leave this issue open and use it as a place to track that work.

Thank you again!

**Comment by bf-tomnomnom** [comment]
Hey @michaellcader! Thank you again for raising a detailed issue and for your patience.

This should be fixed as of #15 being merged. Please let us know if you encounter more issues!
